lib/attachProof.js

```diff
@@ -58,15 +60,15 @@ async function attachProofs(operation, {
     operation = await attachTicketServiceProof({client, ...operation});
   }
   // get invocation target from the record/recordPatch in the operation
-  const invocationTarget = _getInvocationTarget({operation});
+  const target = invocationTarget || _getInvocationTarget({operation});
```

Comment:
Seems like you could just do: `invocationTarget = invocationTarget || _getInvocationTarget({operation})` and then you don't need the extra var or the extra syntax below.

Reply:
As it looks like both you @davidlehn are ok with parameter reassignment provided it is safe, I made this commit: https://github.com/veres-one/did-veres-one/pull/71/commits/1885c1f81bcb7f0506b18945a3c0ed96104f6055 testing this and then merging.